spinnaker/types/webhooks.go

```diff
@@ -22,4 +22,34 @@ type Content struct {
 	ExecutionID string    `json:"executionId"`
 	StartTime   Timestamp `json:"startTime"`
 	EndTime     Timestamp `json:"endTime"`
+	Execution   Execution `json:"execution,omitempty"`
+}
+
+// Execution represents an execution context for a spinnaker event
+type Execution struct {
+	Type             string         `json:"type"`
+	ID               string         `json:"id"`
+	Application      string         `json:"application"`
+	StartTime        Timestamp      `json:"startTime"`
+	EndTime          Timestamp      `json:"endTime"`
+	Name             string         `json:"name"`
+	Canceled         bool           `json:"cancelled"`
+	CancelledBy      string         `json:"cancelledBy,omitempty"`
+	PipelineConfigID string         `json:"pipelineConfigId"`
+	Status           string         `json:"status"`
+	Trigger          Trigger        `json:"trigger,omitempty"`
+	Authentication   Authentication `json:"authentication,omitempty"`
+	Stages           []interface{}  `json:"stages,omitempty"`
```

Comment:
it might be interesting to see more of the stage data in the future, but I've kinda glossed over it for now.